certified-connectors/Maximizer/apiProperties.json

```diff
@@ -0,0 +1,518 @@
+              "uiDefinition": {
+                "constraints": {
+                  "required": "true",
+                  "tabIndex": 3,
+                  "allowedValues": [
+                    {
+                      "text": "default",
+                      "value": "uk1.maximizercrmlive.com"
+                    }
+                  ]
+                },
+                "displayName": "Host",
+                "description": "Select default to continue",
+                "tooltip": "Save your notes about this connection."
+              }
+            },
+            "token:customBasePath": {
+              "type": "string",
+              "uiDefinition": {
+                "constraints": {
+                  "required": "true",
+                  "tabIndex": 3,
+                  "allowedValues": [
+                    {
+                      "text": "default",
+                      "value": "/intMSFlows"
+                    }
+                  ]
+                },
+                "displayName": "Integration path",
+                "description": "Select default to continue",
+                "tooltip": "Save your notes about this connection."
+              }
+            },
+            "token:customClientId": {
+              "type": "string",
+              "uiDefinition": {
+                "constraints": {
+                  "required": "true",
+                  "tabIndex": 3,
+                  "allowedValues": [
+                    {
+                      "text": "default",
+                      "value": "{{OAuth2UKcustomClientId}}"
+                    }
+                  ]
+                },
+                "description": "Select default to continue",
+                "displayName": "Client Id",
+                "tooltip": "Save your notes about this connection."
+              }
+            },
+            "token:customClientSecret": {
+              "type": "string",
+
+              "uiDefinition": {
+                "constraints": {
+                  "required": "true",
+                  "tabIndex": 3,
+                  "allowedValues": [
+                    {
+                      "text": "default",
+                      "value": "{{OAuth2UKcustomClientSecret}}"
+                    }
+                  ]
+                },
+                "description": "Select default to continue",
+                "displayName": "Client Secret",
+                "tooltip": "Save your notes about this connection."
+              }
+            }
+          }
+        }
+      ]
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Set host for connection",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('token:customHost')@connectionParameters('token:customBasePath')"
+        }
+      }
+    ],
+    "publisher": "David Chan"

```

Comment:
@MaximizerIntegration Publisher should be the company name in case of verified publishers. it should be the same with the stackOwner.
please add
    "stackOwner": "Name of the owner of the service or product."